pilot/pkg/model/push_context_test.go

```diff
@@ -565,6 +570,9 @@ func TestWasmPlugins(t *testing.T) {
 			name: "ingress",
 			node: &Proxy{
 				ConfigNamespace: "other",
+				Labels: map[string]string{
```

Comment:
can we make https://github.com/istio/istio/blob/db637b3e4d3c09da0f8fc8f3222231e1834668ed/pilot/pkg/networking/core/v1alpha3/fake.go#L209 handle this so we don't need to change every test to repeat the labels multiple times?

Reply:
have to take a look

Reply:
We could, but ConfigGenTest.SetupProxy require more params

Reply:
I am not sure what you mean? I just mean that we should have users of SetupProxy only set labels once and then we compute them (via SetWorkloadLabels?) for them automatically so they do not need to remember to set 2 sets of the same label

Reply:
I mean many ut are not using `ConfigGenTest`

SetWorkloadLabels is triggered by xds request, xds push, right?

Here it is the ut, which will not call `SetWorkloadLabels`. Do you mean to explicitly call it in `SetupProxy`?

And we transit all tests to use `ConfigGenTest`. We should file a issue to track it, WDYT

Reply:
Ah I see, misunderstood. thanks. I think `model` cannot use ConfigGenTest